README.md

```diff
@@ -118,12 +118,88 @@ The `data` key will contain any resource data received from the API and the `met
 
 ##### meta
 
-* `next`: For any paged resource, `next` will be a cursor to supply for the next page of results. 
-* `previous`: For any paged resource, `previous` will be a cursor to supply for the previous page of results. 
+* `next`: For any paged resource, `next` will be a cursor to supply for the next page of results.
+* `previous`: For any paged resource, `previous` will be a cursor to supply for the previous page of results.
 
 ## Resources
 
-We currently expose two resource to manage, `Orders` and `Accounts`. 
+We currently expose the following resources to manage:
+
+* [`Accounts`](#accounts)
+* [`Merchants`](#merchants)
+* [`Orders`](#orders)
+
+### Accounts
+
+##### All
+
+```javascript
+var client = require('@button/button-client-node')('sk-XXX');
+
+client.accounts.all(function(err, res) {
+    // ...
+});
+```
+
+##### Transactions
+
+Transactions are a paged resource.  The response object will contain properties `meta.next` and `meta.previous` which can be supplied to subsequent invocations of `#transactions` to fetch additional results.
+
+`#transactions` accepts an optional second parameter, `options` which may define the follow keys to narrow results:
+
+###### options
+
+* `cursor`: An API cursor to fetch a specific set of results
+* `start`: An ISO-8601 datetime string to filter only transactions after `start`
+* `end`: An ISO-8601 datetime string to filter only transactions before `end`
+
+```javascript
+var client = require('@button/button-client-node')('sk-XXX');
+
+// without options argument
+//
+client.accounts.transactions('acc-1', function(err, res) {
+    // ...
+});
+
+// with options argument
+//
+client.accounts.transactions('acc-1', {
+  cursor: 'cXw',
+  start: '2015-01-01T00:00:00Z',
+  end: '2016-01-01T00:00:00Z'
+}, function(err, res) {
+    // ...
+});
+```
+
+### Merchants
+
+##### All
+
+###### options
+
+* `status`: Status to filter by. One of ('approved', 'pending', or 'available')
```

Comment:
s/Status/Partnership request status, so it's clear what status we're filtering by :)